ui/src/cloud/apis/orgsettings.ts

```diff
@@ -0,0 +1,16 @@
+import AJAX from 'src/utils/ajax'
+import {OrgSettings} from 'src/types'
+
+export const getOrgSettings = async (orgID: string): Promise<OrgSettings> => {
+  try {
+    const {data} = await AJAX({
```

Comment:
please use `fetch` instead of AJAX

Reply:
Here AJAX is just an alias for axios. Is that okay to use or are we moving away from axios as well?

Reply:
🎉

Reply:
>please use fetch instead of AJAX
why are we moving away from axios to fetch?  do people prefer the fetch api? i tend to find it tedious and verbose to send requests and handling responses is a bit confusing and prone to be buggy, especially if some responses send text and some send json. was there ever a discussion about this?

Reply:
My guess is that axios is a dependency while fetch is built into all the major browsers. And also, that we want to put the nail into the coffin of IE users.